command/kv_helpers.go

```diff
@@ -119,7 +119,11 @@ allow read access to that path in the format 'mount-path/data/foo', not just 'mo
 	return mountPath, 1, nil
 }
 
-func isKVv2(path string, client *api.Client) (string, bool, error) {
+func isKVv2(path string, client *api.Client, kvVersion int) (string, bool, error) {
+	if kvVersion != 0 {
+		return strings.SplitN(path, "/", 2)[0], kvVersion == 2, nil
```

Comment:
What's the reason behind `SplitN` here? Wouldn't `strings.Split(path, "/")[0]` be equivalent?